.nvmrc

```diff
@@ -1 +1 @@
-v12.22.3
+v14.17.0
```

Comment:
@creed-victor was any dependency breaking? I think I've seen that but I want to be sure the reason behind this change

Reply:
Yep, but can't remember which one

Reply:
no worries